docs/configuration/downstream/pull_from_upstream.md

```diff
@@ -9,7 +9,15 @@ A dist-git only job that opens a new dist-git pull request in
 src.fedoraproject.org when a new upstream release happens using a notification
 from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
 for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
-`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. This behaviour can be customized
+`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. 
+
+The `pull_from_upstream` job will close the Bugzillas opened by the Upstream Release Monitoring on your behalf 
+through the [Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html#associate-bugs-to-automatic-updates). 
```

Comment:
Correct me if I'm wrong, but I think this is more accurate:

```suggestion

Bodhi updates created by the `bodhi_update` job as well as [automatic Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html) will have this Bugzilla assigned and it will be closed when they reach stable.
```